tests/backup/backup_test.go

```diff
@@ -188,6 +188,7 @@ var _ = Describe("{BackupCreateKillStoreRestore}", func() {
 			//ValidateApplications(contexts)
 		})
 
+		time.Sleep(time.Minute * 20)
```

Comment:
@vinayakshnd can you sync up with Gleb on why the validate applications was commented above?

Reply:
Sure, main goal of that sleep is wait for IO to happen.